about.html

```diff
@@ -1,12 +1,211 @@
+          />
+          <span class="line"></span>
+        </div>
+        <p class="intro-text about-intro-text text-center">
+          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Lorem ipsum
+          dolor sit, amet consectetur adipisicing elit. Quas, qui. Lorem ipsum
+          dolor sit amet consectetur adipisicing elit. Quae praesentium odit nam
+          commodi voluptatibus iusto consequatur? Esse at placeat ab consequatur
+          impedit aspernatur, ipsam, architecto sapiente porro repellendus
+          libero ut!
+        </p>
+
+        <div class="event-date d-flex flex-column align-items-center">
+          <p class="text w-75 text-center">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Non,
+            excepturi officia voluptate possimus impedit natus enim ad? Ex,
+            minus aliquid!
+          </p>
+          <h3>WebDevBootCamp2022@Douala</h3>
+        </div>
+      </section>
+
+      <section
+        class="my-logo-section container-fluid d-flex flex-column gap-4 p-md-5 p-2 p-lg-5 justify-content-center align-items-center"
+      >
+        <div class="title d-md-flex flex-md-column align-items-center gap-3">
+          <h3 class="text-center">
+            Creative Commons <br class="d-md-none" />Global Summit 2015 Logo
+          </h3>
+          <span class="line"></span>
+          <p class="text text-center">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Non,
+            excepturi officia voluptate possimus impedit natus enim ad? Ex,
+            minus aliquid!
+          </p>
+        </div>
+
+        <div class="my-about-logo d-flex justify-content-center">
+          <img
+            src="images/Ccsummit2015_logo.png"
+            alt="CC-logo"
+            class="cc-image"
+          />
+        </div>
+      </section>
+
+      <section
+        class="CC-past-events container-fluid d-flex flex-column gap-4 p-md-5 p-2 p-lg-5 justify-content-center align-items-center"
+      >
+        <div
+          class="title d-md-flex flex-column gap-2 justify-content-center align-items-center"
+        >
+          <h3 class="text-center">See the past Wed Development Bootcamp</h3>
+          <span class="line"></span>
+          <p class="text text-center">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Non,
+            excepturi officia voluptate possimus
+          </p>
+        </div>
+        <div class="CC-logo d-flex flex-md-row flex-column gap-4">
+          <a href="" class="about-hover"
+            ><img
+              src="images/about_2013_hover.png"
+              alt="about 2013"
+              class="w-100"
+            />
+          </a>
+
+          <a href=" class" class="about-hover"
+            ><img
+              src="images/about_2011_hover.png"
+              alt="about 2011"
+              class="w-100"
+          /></a>
+        </div>
+      </section>
+
+      <!--Footer-->
```

Comment:
- [ ] Above the footer there is supposed to be a partner section for both the mobile and desktop view, kindly add it as it is part of the design.

Your design
![image](https://user-images.githubusercontent.com/81932478/190192788-d0a5b4f7-c0de-4f17-86db-cae1e06cf7c2.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/190192956-4914c2c1-6288-4387-b5ae-be5462b59b82.png)

- [ ] Also fix the color of the footer , it is white and not the color which is currently be used.